main/cropper.js

```diff
@@ -0,0 +1,72 @@
+// Native
+const {format} = require('url');
```

Comment:
`format` is too much of a generic name. It makes for hard to read code. Just import `url` directly or rename it: `const {format: formatUrl} ...`.